public/index.js

```diff
@@ -64,8 +64,12 @@ buttons.forEach((button) => {
       .then((response) => response.json())
       .then((data) => {
         if (resultElement) {
-          resultElement.innerHTML = `<p>${data.choices[0].message.content}</p>`;
-          
+          const textContent = data.text.choices[0].message.content;
+          const imageUrl = data.image.data[0].url;
+          resultElement.innerHTML = `
```

Comment:
Created variables for text and image that can then be called via string literal to create <P> for text and <img> for image. @josueJURE